### PR TITLE
chore: Refresh Azure page

### PR DESCRIPTION
**Pull request by steverydz**
## Done
- Updated the /azure page

## QA
- Go to https://ubuntu-com-14456.demos.haus/azure
- Compare to [copy doc](https://docs.google.com/document/d/1GUlSATrqZmGrsk4DhP9W0s-gcOuFxpdKVgx7YE4g780/edit?tab=t.0)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-16058

### CODE REVIEW

**Comment by steverydz** [comment]
@Sophie-32 @mattea-turic This is now ready for review

**Comment by mattea-turic** [comment]
Hi Steve! Thanks for making the changes I mentioned earlier; I've just noticed two more things:

- hrs that span the entire horizontal should be default colour, not muted
- hero image I uploaded is a little pixelated; I'll ask the pm for the original files so you can swap it in when we get it

I'll +1 since these are pretty small things, and then when we receive the new image, this can be updated

**Comment by mattea-turic** [comment]
@steverydz I've added a higher resolution hero image to the [same assets server](https://assets.ubuntu.com/manager?tag=azure-overview-assets). You can find it [directly here](https://assets.ubuntu.com/v1/1990c79f-canonical-public-cloud-microsoft-illustration.png) too. 

TIA for when you have time to update it!

**Comment by steverydz** [comment]
Thanks @mattea-turic. Those changes have all been made.

File: templates/azure/index.html [hunk 1]
```diff
@@ -116,64 +82,52 @@ <h2>Choose the right Ubuntu for you</h2>
       <div class="col-6">
         <hr class="p-rule--highlight" />
```
Comment (petesfrench) [inline templates/azure/index.html:83]:
```suggestion
        <hr class="p-rule--highlight u-no-margin--bottom" />
```

File: templates/azure/index.html [hunk 2]
```diff
@@ -116,64 +82,52 @@ <h2>Choose the right Ubuntu for you</h2>
...
           <hr class="p-rule--muted" />
           <p>
             <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts"
-               class="p-button">Launch Ubuntu 24.04 LTS</a>
+               class="p-button">Launch Ubuntu on Azure</a>
           </p>
         </div>
       </div>
       <div class="col-6">
         <hr class="p-rule--highlight" />
```
Comment (petesfrench) [inline templates/azure/index.html:105]:
```suggestion
        <hr class="p-rule--highlight u-no-margin--bottom" />
```

File: templates/azure/index.html [hunk 3]
```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
...
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Secure during compute</li>
+          <li class="p-list__item is-ticked">Enables zero trust</li>
+          <li class="p-list__item is-ticked">Process sensitive workloads without worry</li>
+          <li class="p-list__item is-ticked">Available on Ubuntu Pro for added security or the free versions of Ubuntu</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-confidential-vm-jammy?tab=Overview" class="p-button">Launch an Ubuntu Pro CVM now</a></p>
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview" style="margin-right: 1rem">Launch an Ubuntu CVM&nbsp;&rsaquo;</a>
+        <a href="/confidential-computing">Learn more about CVMs&nbsp;&rsquo;</a>
```
Comment (petesfrench) [inline templates/azure/index.html:255]: You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

File: templates/azure/index.html [hunk 4]
```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
...
       <div class="col">
         <h2>Optimise your cloud journey with Canonical</h2>
       </div>
       <div class="col">
-        <p>
-          Canonical works with the top enterprises and innovators in the world to define and execute their cloud journeys. Migration, application redesign, support and optimisation services are some of the ways in which we have helped customers make the most of Ubuntu and Azure together.
-        </p>
+        <p>Canonical works with the top enterprises and innovators in the world to define and execute their cloud journeys. Migration, application redesign, support and optimization services are some of the ways in which we have helped customers make the most of Ubuntu and Azure together.</p>
         <hr class="p-rule--muted" />
         <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
```
Comment (petesfrench) [inline templates/azure/index.html:282]: You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

File: templates/azure/index.html [hunk 5]
```diff
@@ -22,25 +22,20 @@
...
-          </p>
+          <p>Canonical produces official Ubuntu images on Microsoft Azure with continuous updates to ensure security and stability from the moment your machines and containers launch.</p>
+          <p>Microsoft works closely with Canonical to optimize Ubuntu images on Microsoft Azure and ensure Ubuntu supports the latest cloud features as these are released. Today, Ubuntu powers a large number of mission-critical workloads on Azure.</p>
+          <p>Beyond Ubuntu, Canonical also offers Charmed Apps on Microsoft Azure, enabling hybrid environments.</p>
         </div>
         <hr class="p-rule--muted" />
-        <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview" class="p-button--positive">Launch Ubuntu 24.04 on Azure</a>
+        <a href="/azure/contact-us" class="js-invoke-modal">Talk to us about Ubuntu on Azure&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/azure/index.html:31]: You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

File: templates/azure/index.html [hunk 6]
```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
...
 
-  <section class="p-section--deep">
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--highlight" />
+        <h2 class="p-muted-heading">Hear from our partners</h2>
+      </div>
+      <div class="row--25-75 p-section--shallow">
+        <div class="col">
```
Comment (petesfrench) [inline templates/azure/index.html:190]: You could use the [Vanilla quote macro](https://vanillaframework.io/docs/patterns/quote-wrapper#without-heading) for these (I think, I haven't tried it myself)

File: templates/azure/index.html [hunk 7]
```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
...
+          }}
+        </div>
+        <div class="col">
+          <div class="row">
+            <hr class="p-rule--muted" />
+            <div class="col-6">
+              <p class="p-heading--4"><em>"Thanks to the combination of Microsoft Azure’s infrastructure and Canonical Ubuntu's stability and security, businesses can efficiently deploy agile applications, leverage AI and machine learning, and ensure regulatory compliance. Our collaboration with Canonical underscores a commitment to delivering flexible, innovative solutions via Azure Marketplace that drive digital transformation and operational excellence for organizations."</em></p>
+            </div>
+            <div class="col-3">
+              <p class="u-no-margin--bottom"><strong>Yvonne Muenc</strong></p>
```
Comment (petesfrench) [inline templates/azure/index.html:206]:
```suggestion
              <p class="u-no-margin--bottom"><strong>Yvonne Muench</strong></p>
```

File: templates/azure/index.html [hunk 8]
```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
...
+  </section>
+
+  <section class="p-section">
     <div class="row--50-50">
       <hr class="p-rule--muted" />
+      <div class="col">
+        <h2>Secure Data at Run-Time with Confidential Computing on Azure</h2>
+      </div>
+      <div class="col">
+        <p>If you're running critical workloads and need to process sensitive data, security at system memory is essential. With confidential computing, your workloads are  encrypted at run time, and are thus secure from all software vulnerabilities outside the VM perimeter - including the hypervisor and from unauthorized cloud admins. Pair confidential VMs with Ubuntu Pro for security in and out of your VM.</p>
```
Comment (petesfrench) [inline templates/azure/index.html:245]:
```suggestion
        <p>If you're running critical workloads and need to process sensitive data, security at system memory is essential. With confidential computing, your workloads are  encrypted at run time, and are thus secure from all software vulnerabilities outside the VM perimeter &ndash; including the hypervisor and from unauthorized cloud admins. Pair confidential VMs with Ubuntu Pro for security in and out of your VM.</p>
```

File: templates/azure/index.html (hunk 3 above)
Comment (petesfrench) [inline templates/azure/index.html:255]:
```suggestion
        <a href="/confidential-computing">Learn more about CVMs&nbsp;&rsaquo;</a>
```

**Review by petesfrench — CHANGES_REQUESTED** [review]

File: templates/azure/index.html (hunk 5 above)
Comment (steverydz) [reply templates/azure/index.html]: Done

File: templates/azure/index.html [hunk 9]
```diff
@@ -116,64 +82,52 @@ <h2>Choose the right Ubuntu for you</h2>
...
+            <li class="p-list__item is-ticked">No automated compliance and hardening</li>
+            <li class="p-list__item is-ticked">Patching requires a reboot</li>
+            <li class="p-list__item is-ticked">Community and update level support</li>
           </ul>
           <hr class="p-rule--muted" />
           <p>
             <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts"
-               class="p-button">Launch Ubuntu 24.04 LTS</a>
+               class="p-button">Launch Ubuntu on Azure</a>
           </p>
```
[the remark this answers is not in the document]
Comment (steverydz) [reply templates/azure/index.html]: Done

File: templates/azure/index.html (hunk 4 above)
Comment (steverydz) [reply templates/azure/index.html]: Done

File: templates/azure/index.html (hunk 3 above)
Comment (steverydz) [reply templates/azure/index.html]: Doesn't work in this case because of the top link

Comment (steverydz) [reply templates/azure/index.html]: Done

**Review by petesfrench — APPROVED** [review]

**Comment by akbarkz** [comment]
@steverydz may I know if there is a reason why this PR is not merged yet?

**Pull request merged**